Dockerfile

```diff
@@ -2,31 +2,23 @@
 # the Google Cloud Platform.
 
 # Install pipelines tool.
-FROM golang:1.10
-ENV GOPATH /go
+FROM golang:1.11
 RUN go get github.com/googlegenomics/pipelines-tools/pipelines
 
-
 FROM google/cloud-sdk:alpine
 
 # Copy pipelines tool from the previous step.
-ENV GOPATH /go
-COPY --from=0 /go/ $GOPATH
-ENV PATH $PATH:$GOPATH/bin
+COPY --from=0 /go/bin/pipelines /usr/bin
+
+ARG commit_sha
```

Comment:
I don't see where you are using this?

Reply:
Again copied from VariantTransforms. I think the idea is this environment variable can be used to map the image back to the code it was build with. This command will show all environment variables:
`docker inspect gcr.io/...`

However, now when I checked it for our docker image `COMMIT_SHA` was empty string. I need to check again why it's missing.

Reply:
COMMIT_SHA is filled in by cloud build when it builds from a git tag.  But you aren't using it here.

We use it in VT because VT actually needs to test the same image that was built since it re-invokes itself using pipelines API.  We don't need it here.

Reply:
Removed